### DISCUSSION TITLE
Showing off: What I build with FluentUI, Part 2

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#18862] [Show and tell]
Hi,

I would like to demonstrate yet another part of the application I am building with FluentUI. The YouTube video below, is a demonstration of uploading (large) images into the application and the ability to preview/edit the pictures using a `Panel` component.

Although with a completely different implementation,  the `TilesList` component is a custom component visually modeled after the `TilesList` component from the `@fluentui/react-experiments` package.

I have used the `react-dropzone` within a `Dialog` component for uploading the images. 
The backend is developed using `NodeJS` and `PostgreSQL`. 

This application is being developed for managing a large number actor profiles to be able to create castings for movies and TV commercials.

Thanks for your time.

[![Uploading and ImageTile](https://gist.github.com/blendsdk/1dcad051db496ce2eefed3780ad9cf49/raw/413a7c5eaf880e5c9c904d77b9783b943076e607/uploading_cover.png)](https://www.youtube.com/watch?v=3YKNKglQuJU)

**Comment by khmakoto** [comment]
This is awesome! Thanks for showing us these snippets of what you're doing, they really enlighten our day!
